app/views/idv/cac/welcome.html.erb

```diff
@@ -0,0 +1,97 @@
+<h1 class="h3 my0">
+  <%= t('cac_proofing.headings.welcome') %>
+</h1>
+
+<p class="mt-tiny mb3">
+  <%= t('cac_proofing.info.welcome') %>
+</p>
+
+<h1 class="h3 mb2">
+  <%= t('cac_proofing.instructions.welcome') %>
+</h1>
+
+<ul class="list-reset">
+  <li class="pt2 pb1">
+    <div class="inline-block mr2 mt1 align-top circle circle-number bg-blue white">
+      1
+    </div>
+    <div class="mr1 inline-block">
+      <div class="h1 inline-block bold">
+        <%= t('cac_proofing.instructions.bullet1') %>
+      </div>
+      <br/>
+      <%= t('cac_proofing.instructions.text1') %>
+    </div>
+  </li>
+  <li class="pt2 pb1">
+    <div class="inline-block mr2 mt1 align-top circle circle-number bg-blue white">
+      2
+    </div>
+    <div class="mr1 inline-block">
+      <div class="h1 inline-block bold">
+        <%= t('cac_proofing.instructions.bullet2') %>
+      </div>
+      <br/>
+      <%= t('cac_proofing.instructions.text2') %>
+    </div>
+  </li>
+  <li class="pt2 pb1">
+    <div class="inline-block mr2 mt1 align-top circle circle-number bg-blue white">
+      3
+    </div>
+    <div class="mr1 inline-block">
+      <div class="h1 inline-block bold">
+        <%= t('cac_proofing.instructions.bullet3') %>
+      </div>
+      <br/>
+      <%= t('cac_proofing.instructions.text3') %>
+    </div>
+  </li>
+  <li class="pt2 pb1">
+    <div class="inline-block mr2 mt1 align-top circle circle-number bg-blue white">
+      4
+    </div>
+    <div class="mr1 inline-block">
+      <div class="h1 inline-block bold">
+        <%= t('cac_proofing.instructions.bullet4') %>
+      </div>
+      <br/>
+      <%= t('cac_proofing.instructions.text4') %>
+    </div>
+  </li>
+  <li class="pt2 pb1">
+    <div class="inline-block mr2 mt1 align-top circle circle-number bg-blue white">
+      5
+    </div>
+    <div class="mr1 inline-block">
+      <div class="h1 inline-block bold">
+        <%= t('cac_proofing.instructions.bullet5') %>
+      </div>
+      <br/>
+      <%= t('cac_proofing.instructions.text5') %>
+    </div>
+  </li>
+</ul>
+
+<%= validated_form_for(:cac, url: url_for, method: 'PUT',
+                       html: { autocomplete: 'off', role: 'form', class: 'mt2' }) do %>
+  <br/>
+  <div class="mt0">
+    <button type="submit" class="btn btn-primary btn-wide sm-col-6 col-6">
+      <%= t('cac_proofing.buttons.get_started') %>
+    </button>
+  </div>
+<% end %>
+
+<br/>
+
+<div class="mt2 pt1 border-top">
+</div>
+
+<% if user_fully_authenticated? %>
+  <%= link_to cancel_link_text, account_path, class: 'h5' %>
+<% else %>
+  <%= link_to t('two_factor_authentication.choose_another_option'), two_factor_options_path %>
+<% end %>
```

Comment:
Hm, I'd guess the `div` was intended to have been a wrapper for this content? Seems like it'd be more appropriate markup, though this does appear to be the correct 1-to-1 conversion, so also fine avoiding the risk of making changes 🤷

Reply:
Yeah it wasn't clear whether it was intended as kind of a blank spacer with a border or what so I left it